packages/typespec-azure-resource-manager/src/commontypes.private.decorators.ts

```diff
@@ -127,6 +127,12 @@ export function $armCommonDefinition(
   storeCommonTypeRecord(context, entity, "definitions", definitionName, version, referenceFile);
 }
 
+/**
+ * Specify the ARM commont type version reference for a particular spec version or namespace.
+ * @param {DecoratorContext} context DecoratorContext object
+ * @param {enumType} entity Decorator target type. Must be `Model`
+ * @returns {void}
```

Comment:
nit but I don't think its ncessary to include the tyupes in the jsdoc with typescript

Reply:
You meant the @param?

Reply:
oh wait I misread. Just remove the wrapping `{}`
usualy in JS doc the syntax is
```
@param foo {Type} Dec of param
```

but in tsdoc is should just be 
```
@param foo Dec of param
```